pr_testing/test_multiple_prs.sh

```diff
@@ -1157,11 +1158,20 @@ if [ "X$BUILD_OK" = Xtrue -a "$RUN_TESTS" = "true" ]; then
     DO_GPU_TESTS=true
     mark_commit_status_all_prs 'unittests/gpu' 'pending' -u "${BUILD_URL}" -d "Waiting for tests to start"
   fi
+  if [ $(echo ${ENABLE_BOT_TESTS} | tr ',' ' ' | tr ' ' '\n' | grep '^HLT_P2_TIMING$' | wc -l) -gt 0 ] ; then
+    if [ $(echo ${ARCHITECTURE}   | grep "_amd64_" | wc -l) -gt 0 ] ; then
+      if [ -e ${CMSSW_CVMFS_PATH}/src/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh ]; then
```

Comment:
please use `$CMSSW_RELEASE_BASE`  instead of `${CMSSW_CVMFS_PATH}` ( not that CMSSW_CVMFS_PATH is not accessible when this script is running. Output of this script gets deployed under CMSSW_CVMFS_PATH)